From 6d63ad8708aed8a1adc9a39dad536a0b081488ae Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Tue, 3 Feb 2015 16:09:58 -0600
Subject: [PATCH 06/13] ARM: dts: AM4372: Correct node addresses for PRU-ICSS1
 PRU cores

The addresses for the PRU cores in PRU-ICSS on AM437x differs from
that of the corresponding cores on AM33xx family of SoCs. The node
names are incorrectly using the addresses from the AM33xx PRU cores,
even though the reg properties are defined correctly. Fix this typo
in the node addresses.

Signed-off-by: Suman Anna <s-anna@ti.com>
---
 arch/arm/boot/dts/am4372.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/am4372.dtsi b/arch/arm/boot/dts/am4372.dtsi
index 236729d..0bc457a 100644
--- a/arch/arm/boot/dts/am4372.dtsi
+++ b/arch/arm/boot/dts/am4372.dtsi
@@ -719,7 +719,7 @@
 			#size-cells = <1>;
 			ranges;
 
-			pru0: pru@4a334000 {
+			pru0: pru@54434000 {
 				compatible = "ti,pru-rproc";
 				reg = <0x54434000 0x3000>,
 				      <0x54422000 0x400>,
@@ -728,7 +728,7 @@
 				mboxes = <&mailbox &mbox_pru0>;
 			};
 
-			pru1: pru@4a338000 {
+			pru1: pru@54438000 {
 				compatible = "ti,pru-rproc";
 				reg = <0x54438000 0x3000>,
 				      <0x54424000 0x400>,
-- 
1.7.9.5

